### PR TITLE
Feat/query

### PR DESCRIPTION
**Pull request by felippepuhle**
- [x] implementation
- [x] tests
- [x] docs

### CODE REVIEW

**Review by AndreiCalazans — DISMISSED** [review]
Nice

File: README.md [hunk 1]
```diff
@@ -60,7 +60,38 @@ ReactDOM.render(
...
+## Examples
+
+Let's start making a simple [query](docs/hocs/query.md):
+
+```javascript
+import { graphql } from 'react-relay';
+import { query } from 'react-releasy';
+
+const MyComponent = ({ environment }) => {
+  if (error) {
```
Comment (rodrigooler) [inline README.md]: You did not declare this prop

File: README.md [hunk 2]
```diff
@@ -60,7 +60,38 @@ ReactDOM.render(
...
+    return `Error: ${error.message}`;
+  }
+
+  if (isFetching) {
```
Comment (rodrigooler) [inline README.md:76]: Same

File: README.md [hunk 3]
```diff
@@ -60,7 +60,38 @@ ReactDOM.render(
...
+    return 'Loading...';
+  }
+
+  return `My name is ${me.name}`;
```
Comment (rodrigooler) [inline README.md:80]: Same

File: docs/hocs/query.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+An higher-order component to [Relay QueryRenderer](https://facebook.github.io/relay/docs/en/query-renderer.html).
+
+## Usage
+
+```javascript
+import { graphql } from 'react-relay';
+import { query } from 'react-releasy';
+
+// ... more code here
```
Comment (rodrigooler) [inline docs/hocs/query.md:11]: Detail here the code to be clearer

Comment (felippepuhle) [reply docs/hocs/query.md:11]: this won't be necessary for now, we have a lot of examples at the root `README.md` and there's nothing different from a simple React component

**Review by rodrigooler — APPROVED** [review]
Implementation makes excellent work sense, I put some comments to improve the doc.

**Pull request merged**